core/commonTest/src/kotlinx/serialization/SerializersLookupEnumTest.kt

```diff
@@ -74,6 +74,7 @@ class SerializersLookupEnumTest {
     }
 
     @Test
+    @Ignore // @Polymorphic enum doesn't make sense and is not supported in intrinsics
```

Comment:
We need to chose what to do with `@Polymorphic enum`. My suggestion is to remove this test completely.
Optionally — add diagnostic that prohibits `@Polymorphic` on enums and objects

Reply:
I suggest removing it. The diagnostic would be a good next step -- could you please file a ticket?